.idea/workspace.xml

```diff
@@ -1,130 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why is this deleted?

Reply:
The file shouldn't be uploaded, according to [the official .gitignore by JetBrains](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore)